src/movies.js

```diff
@@ -1,22 +1,64 @@
 // Iteration 1: All directors? - Get the array of all directors.
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors.
 // How could you "clean" a bit this array and make it unified (without duplicates)?
-function getAllDirectors(moviesArray) {}
+function getAllDirectors(moviesArray) {
+    return movies.map((value) => value.director)
```

Comment:
The function is trying to access the 'movies' property, but it should be accessing the 'moviesArray' parameter that is passed into the function.

`return moviesArray.map((value) => value.director)`